notebooks/agent_with_human_in_the_loop.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<p>What is the difference between this prompt and the previous one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>